buck2.yaml

```diff
@@ -1,8 +1,8 @@
 package:
   name: buck2
-  # When bumping, check that the rust version is still correct.
-  version: 0.0_git20230706
-  epoch: 2
+  # When bumping, bump the tag below
+  version: 20230815
```

Comment:
I would stick with 0_git20230815 myself.

Reply:
This is a real tag now, so stick with git anyway?

Reply:
The tag is 2023-08-15, not 20230815, so it needs to be adjusted to our format anyway.  I would say to go with `_git` because it will ensure that it gets ranked below 0.1 should they choose to do a real release rather than a snapshot in the future.